src/components/Tabbar/index.js

```diff
@@ -1,44 +1,45 @@
 import React from 'react';
-import {View, TouchableOpacity, StyleSheet} from 'react-native';
-import Animated from 'react-native-reanimated';
-import {fonts} from '../../utils/fonts';
+import {View, TouchableOpacity, StyleSheet, Text} from 'react-native';
+import {fonts, normalizeFontSize} from '../../utils/fonts';
 import {COLORS} from '../../utils/theme';
+import dimen from '../../utils/dimen';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `COLORS` import statement is duplicated. This could lead to confusion and potential errors in the future. Please remove one of them.
```diff
- import {fonts, normalizeFontSize} from '../../utils/fonts';
-  import {COLORS} from '../../utils/theme';
+ import {fonts, normalizeFontSize} from '../../utils/fonts';
+ import {COLORS} from '../../utils/theme';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->